ph5/utilities/validation.py

```diff
@@ -366,7 +366,9 @@ def check_lat_lon_elev(station):
                       % station['location/Y/value_d'])
     if station['location/Y/units_s'] in [None, '']:
         warnings.append("No Station location/Y/units_s value found.")
-
+    if station['location/Z/units_s'] in ['unknown', 'UNKNOWN', 'Unknown']:
```

Comment:
To cover all combination of unknown, I think this line should be written like this:
`if station['location/Z/units_s'].lower() == 'unknown':`